build/webpack/webpack.config.js

```diff
@@ -0,0 +1,170 @@
+/*
+This is just porting all the configs from all the seperate transform files and
+having one large standard one. This should make it easier to update to webpack 2
+As well as just making it more accessible without having to dive around all the
+transforms. Would prefer to have a well commented file like Origami:
+*/
+
+const path =require('path');
+const glob = require('glob');
+const BowerWebpackPlugin = require('bower-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const autoprefixer = require('autoprefixer');
+
+
+const handlebarsConfig = () => {
+	const extraHelperDirs = glob.sync('**/node_modules/@financial-times/**/handlebars-helpers')
+		.map(dir => path.resolve(dir));
+	return {
+		debug: false, // set to true to debug finding partial/helper issues
+		extensions: ['.html'],
+		helperDirs: [
+			path.resolve('./node_modules/@financial-times/n-handlebars/src/helpers'),
```

Comment:
Why not use `require.resolve` instead of `path.resolve`?  https://nodejs.org/dist/latest-v6.x/docs/api/globals.html#globals_require_resolve